cfgov/v1/atomic_elements/molecules.py

```diff
@@ -51,6 +51,7 @@ class ImageText2575(blocks.StructBlock):
     image = atoms.ImageBasic()
     links = blocks.ListBlock(atoms.Hyperlink(), required=False)
     has_rule = blocks.BooleanBlock(required=False)
+    should_link_image = blocks.BooleanBlock(default=False, required=False)
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2554/commits/1b253fecb1e55cb4b92f5d2d407fd05149b64e3c